src/ORM/Query.php

```diff
@@ -1275,4 +1275,22 @@ protected function _decorateResults($result)
 
         return $result;
     }
+
+    /**
+     * Selects all fields in the schema with the exception of the fields you pass
+     * in the $excludedFields parameter.
+     *
+     * @param array $excludedFields fields you don't want to select
+     * @param bool $overwrite whether to reset fields with passed list or not
+     * @return $this
+     */
+    public function selectAllExcept(array $excludedFields = [], $overwrite = false)
```

Comment:
@lorenzo If I change the first param to an Association or TableSchema, it's no longer possible to do something like:

```php
$this->Articles->find()
   ->selectAllExcept(['modified', 'created']);
```

Then it becomes something like:

```php
$this->Articles->find()
   ->selectAllExcept($this->Articles, ['modified', 'created']);
```
In this case the Table parameter is redundant. 

Maybe we should not make the Table/Association the first param, but an optional parameter.

```php
$this->Articles->find()
   ->selectAllExcept(['modified', 'created'], $this->Articles);
```